content/news/XXXX-XX-XX-mixxx-2-3-6-released.md

```diff
@@ -0,0 +1,75 @@
+title: Mixxx 2.3.6 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.6, release announcement
+status: draft
+comments: yes
+
+Welcome the Mixxx release version 2.3.6, the final maintenance release before releasing the upcoming 2.4.0 with exiting new features.
+
+This little bug fix release may appear insignificant, but it's a testament to the rock-solid quality we bring to every user and ensuring a reliable mixing experience.
+
+This release improves among other things some details with looping, allows playing tracks exceeding 6 hours and fixes rare crashes in exceptional situations.
+
+You can download the new version here: https://mixxx.org/download/ There you will also find the 2.4 beta with which you can already try the new features that will be released soon.
```

Comment:
```suggestion
You can download the new version from https://mixxx.org/download/ There, you will also find the 2.4&nbsp;beta to test the new features that will be released soon.
```
nitpick: let's keep "2.4 beta" together. &nbsp; is a hack, ^+space should be preferred AFAIU